src/test/java/com/wikia/webdriver/testcases/infoboxbuilder/InfoboxBuilderTests.java

```diff
@@ -89,42 +92,53 @@ public void deletingAddedComponents() {
         imageComponents,
         builder.deleteImageComponentWithIndex(builder.countImages() - 1).countImages()
     );
+    Assertion.assertEquals(
+        headerComponents,
+        builder.deleteHeaderComponentWithIndex(builder.countHeaders() - 1).countHeaders()
+    );
   }
 
   @Execute(asUser = User.USER)
   public void customizingComponents() {
-    InfoboxBuilderPage builder = new InfoboxBuilderPage();
-
-    String infoboxPreviewBackground = builder.open("customizingComponents")
-        .switchToIFrame()
-        .getBackgroundColor();
-
-    TemplatePage template = builder
-        .open("customizingComponents")
-        .switchToIFrame()
-        .setAndVerifyRowLabelWithIndex(0, "AutomatedTest")
-        .setLongLabelNameAndVerifyBreakLine(1, "AutomatedTestVeryLabelName")
+    TemplatePage template = new InfoboxBuilderPage()
+        .open("InfoboxBuilderCustomizingComponents")
+        .changeHeaderCollapsibilityState(0)
+        .setAndVerifyRowLabel(0, "AutomatedTest")
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "AutomatedTest" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)